omnigibson/object_states/particle_modifier.py

```diff
@@ -102,7 +102,7 @@ def create_projection_visualization(
         # Default to close to singular point otherwise
         source_radius = m.PROJECTION_VISUALIZATION_CONE_TIP_RADIUS
         spread_ratio = projection_radius * 2.0 / projection_height
-        spread = np.ones(3) * spread_ratio * m.PROJECTION_VISUALIZATION_ORIENTATION_BIAS
+        spread = np.ones(3) * spread_ratio * 0.8
```

Comment:
should 0.8 also be a constant defined in the macro?

Reply:
Perhaps? I was thinking about it, but it's not really a macro -- it's more like a magic scaling number that makes the particles sort of follow the correct shape of the cone. I can make it a macro tho!

Reply:
actually i will make it a macro